contributing/development/core_and_modules/core_types.rst

```diff
@@ -103,7 +103,18 @@ which are equivalent to new, delete, new[] and delete[].
 memnew/memdelete also use a little C++ magic and notify Objects right
 after they are created, and right before they are deleted.
 
-For dynamic memory, use Vector<>.
+For dynamic memory, use Godot's ``Vector<>`` or one of its variations.
```

Comment:
`LocalVector` is not a variation of `Vector`. They're two unrelated types. I'd rather phrase it like "Use one of the sequence types" or something like that. `List` has a very different backing technique than both `Vector` and `LocalVector`, but should probably be mentioned here too.